spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/java/org/springframework/cloud/data/module/deployer/cloudfoundry/CloudFoundryClient.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
So is the intent that this code goes away at some point?  If so, can we get a story that outlines that in Jira and includes what we're waiting on?